site/docs/foundations/typography.mdx

```diff
@@ -3,161 +3,36 @@ title: Typography
-  hierarchy.
-</ExampleContainer>
-
-<ExampleContainer type="negative">
-<Image
-  src="/img/foundations/typography-color-heading.svg"
-  alt="Example showing heading colored incorrectly"
-/>
-
-Don’t use colors, other than the primary and secondary options, to emphasize headings, labels, alerts or blocks of text.
+The legacy Salt theme uses the open-source, sans-serif typeface **Open Sans**. It’s the sole typeface for all text styles, with the exception of code which uses the mono space typeface, PT Mono.
 
-</ExampleContainer>
+## Font weight
 
-All text within the design system meets the Web Content Accessibility Guidelines (WCAG) 2.0 level AA requirements to ensure legibility.
+Weight refers to the relative thickness of a font’s stroke. Each type style includes three weights which enable you to create visual hierarchy within the same text size.
 
-WCAG 2.0 requires that the contrast ratio between text and background is at least 4.5:1 for normal-sized text and 3:1 for larger text.
+Please note that in the Salt design system, 'Strong' indicates bolder weight than the default, whereas 'Small' represents a thinner weight, which may differ from traditional terminology. 
 
-<ExampleContainer type="positive">
-  <Image
-    src="/img/foundations/typography-iconography.svg"
-    alt="Example showing color and iconography used to infer meaning"
-  />
-  Use color and iconography with text to help convey additional meaning.
-</ExampleContainer>
+#### Small
 
-<ExampleContainer type="negative">
-  <Image
-    src="/img/foundations/typography-color-meaning.svg"
-    alt="Example showing color used to infer meaning"
-  />
-  Don’t use color alone to infer meaning. Color-blind and low-vision users may
-  not perceive the difference, and screen readers do not announce colors to
-  non-sighted readers.
-</ExampleContainer>
-
-## Line height
+A thinner weight alternative to de-emphasize default weight.
 
-As with most aspects of the design system, type conforms to [density](./density). Each style grows proportionately to align with the elements. Line height and the spaces between blocks also adhere to density.
+#### Default 
 
-Line height controls the space between baselines within a text. The height of a line of text is proportional to the text size.
+The default weight across the system for visual balance.
 
-Line height is set to a multiple of the font size to scale consistently across all styles, weights and densities. The multiple is set to 1.3x. This balances the need for moderate information density with the need to create enough visual separation between lines of text to enable efficient line tracking when reading blocks of text.
+#### Strong
 
-## Weight
-
-Weight refers to the relative thickness of a font’s stroke. Each style includes three weights, enabling you to emphasize or de-emphasize the default style to help create visual hierarchy.
+A stronger weight alternative to emphasize default weight.
 
 <DesignCards>
```

Comment:
Looking at site preview, it's probably better to position text together with the images (like Display section + image below)